File: rolemadelen/programmers/120848_팩토리얼.cpp
int solution(int n) {
    int arr[11] = {1, 1, 2, 6, 24, 120, 720, 5040, 40320, 362880, 3628800};
    for(int i=0; i<11; ++i) {
        if(arr[i] > n) return i-1;
    }
    
    return 10;
}

File: gamsojji/programmers/beginner/120848_팩토리얼.js
function solution(n) {
    var answer = 1;
    var num = 1;
    while(num < n){
        if(팩토리얼(num) <= n && n  < 팩토리얼(num+1)) return num;
        else num++;
    }
}

function 팩토리얼(num){
    let result = 1;
    for(let i = 2; i <= num; i++){
        result *= i;
    }
    return result;
}